### PR TITLE
add centos to supported distros

### PR DESCRIPTION
**Pull request by crunchywelch**
Signed-off-by: Aaron Welch welch@packet.net

### CODE REVIEW

File: hack/install.sh [hunk 1]
```diff
@@ -81,7 +81,7 @@ fi
 
 lsb_dist="$(echo "$lsb_dist" | tr '[:upper:]' '[:lower:]')"
 case "$lsb_dist" in
-	amzn|fedora)
+	amzn|fedora|coreos)
```
Comment (unclejack) [inline hack/install.sh]: This says coreos, not centos.

**Comment by crunchywelch** [comment]
oh hell, sorry, doing too many things at once, let me update

**Comment by jfrazelle** [comment]
but..... centos uses devmapper and cannot use the static binaries we distribute ping @vbatts

**Comment by jfrazelle** [comment]
oh jk this yum installs nm nm

/me hides

**Comment by crunchywelch** [comment]
@jfrazelle :D

**Comment by icecrime** [comment]
SGTM, but would you please mind squashing your two commits?

**Comment by crunchywelch** [comment]
@icecrime done!

**Comment by icecrime** [comment]
Thanks! Ping @tianon for review <3

**Comment by crunchywelch** [comment]
@icecrime @tianon did that jenkins build fail because of something I need to change, or is this good to go and just needs final review?

**Comment by tianon** [comment]
LGTM :+1:

**Comment by icecrime** [comment]
LGTM, thanks!

**Comment by icecrime** [comment]
I'm not sure this needs any docs, but double checking with @moxiegirl! Thanks :-)

**Comment by moxiegirl** [comment]
LGTM

**Pull request merged**